src/raster/mdarray.rs

```diff
@@ -440,13 +451,45 @@ impl<'a> Group<'a> {
             Ok(Attribute::from_c_attribute(c_attribute))
         }
     }
+
+    pub fn dimensions(&self, options: CslStringList) -> Result<Vec<Dimension>> {
+        unsafe {
```

Comment:
Please try to make this `unsafe` block smaller :-).

Reply:
[c482412](https://github.com/georust/gdal/pull/291/commits/c4824127bcb594f53e4c23b2de03b32504fe1d7a)